plat/kvm/x86/setup.c

```diff
@@ -111,7 +111,7 @@ void _ukplat_entry(struct lcpu *lcpu, struct ukplat_bootinfo *bi)
 	/* Print boot information */
 	ukplat_bootinfo_print();
 
-#ifdef CONFIG_HAVE_SMP
+#if defined(CONFIG_HAVE_SMP) && defined(CONFIG_UKPLAT_ACPI)
 	rc = acpi_init();
```

Comment:
Would it make sense to call `acpi_init` even if only `CONFIG_UKPLAT_ACPI` is enabled? As far as I know it can be also used for non-SMP functionality.

Reply:
Yes, it makes perfect sense to want to do that :smiley:. I would prefer we leave it like this (for now :wink:). Instead, I think I should just remove the `UKPLAT_ACPI` from the `Config.uk` and let it only be available by being selected by `CONFIG_HAVE_SMP` and for `UEFI` builds since, at the moment, ACPI is only used for SMP. Indeed, if in the future someone ends up implementing something else like ACPI events handling *wink* *wink* (or maybe even hotplug :hot_face:), this change will definetely be done to be able to have it in non-SMP builds as well.